tox.ini

```diff
@@ -13,9 +13,10 @@ deps =
 commands = pytest {posargs}
 
 [testenv:lint]
-deps = pre-commit~=1.20
-skip_install = true
-commands = pre-commit run --all-files
+extras = lint
+commands =
+    pre-commit run --all-files
+    mypy .
```

Comment:
Won't this mean that mypy runs twice--once in pre-commit and once here?

Reply:
Yes it does. In pre-commit it doesn't have the typehints of dependencies' so the checks aren't as comprehensive. If you want, we can remove mypy from pre-commit. I left it because it can still be useful when making a git commit locally.

Reply:
I'd be fine removing mypy from pre-commit. the `lint` job is already the slowest job in CI at the moment, and I'd hate to make it slower. Also, it's one less thing to manually sync between pre-commit-config.yaml and setup.py.

Reply:
Also, maybe add a comment above this line explaining why we're running mypy outside of pre-commit

Reply:
Removed mypy from pre-commit and made the comment in tox.ini